code/modules/cargo/packs.dm

```diff
@@ -279,6 +279,33 @@
 					/obj/item/clothing/suit/armor/vest)
 	crate_name = "armor crate"
 
+/datum/supply_pack/security/assorted_lens
```

Comment:
Remove this, as cobby said it's sidestepping needing the research which is a bad plan